homeassistant/components/mutesync/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "mütesync",
```

Comment:
Does this need translation? As it seems a brand name, which generally isn't translated

Reply:
No translation needed, it is indeed a brand name.

Reply:
Then remove it. It will fallback automatically to the name in the manifest.

```suggestion
```